spotirec.py

```diff
@@ -337,16 +340,16 @@ def print_blacklist():
     with open(blacklist_path, 'r') as file:
         try:
             blacklist = json.loads(file.read())
-            print('Tracks')
-            print('--------------------------')
+            logging.info('Tracks')
+            logging.info('--------------------------')
             for x in blacklist['tracks'].values():
-                print(f'{x["name"]} by {", ".join(x["artists"])} - {x["uri"]}')
-            print('\nArtists')
-            print('--------------------------')
+                logging.info(f'{x["name"]} by {", ".join(x["artists"])} - {x["uri"]}')
+            logging.info('\nArtists')
+            logging.info('--------------------------')
             for x in blacklist['artists'].values():
-                print(f'{x["name"]} - {x["uri"]}')
+                logging.info(f'{x["name"]} - {x["uri"]}')
         except json.decoder.JSONDecodeError:
-            print('Blacklist is empty')
+            logging.info('Blacklist is empty')
```

Comment:
Would this mean that `spotirec --print blacklist -q would print nothing? I don't see why anyone would use those two arguments together, but these should probably be printed no matter what. Also, line 352 should probably be an error, or warning.

Reply:
Fixed in d3df488, I chose to make them default print - which corresponds with the idea mentioned in last discussion.

Reply:
Great!